spec/support/evm_spec_helper.rb

```diff
@@ -79,33 +85,19 @@ def self.filter_specific_features(hashes, features)
   private_class_method :filter_specific_features
 
   def self.seed_admin_user_and_friends
-    guid, server, zone = create_guid_miq_server_zone
-
-    admin_role = FactoryGirl.create(:miq_user_role,
-      #:name       => "EvmRole-administrator",
-      :name       => "EvmRole-super_administrator",
-      :read_only  => true,
-      :settings   => nil
+    create_guid_miq_server_zone
+
+    FactoryGirl.create(:user,
+      :name       => "Administrator",
+      :email      => "admin@example.com",
+      :password   => "smartvm",
+      :userid     => "admin",
+      :settings   => {"Setting1"  => 1, "Setting2"  => 2, "Setting3"  => 3 },
+      :filters    => {"Filter1"   => 1, "Filter2"   => 2, "Filter3"   => 3 },
+      :evm        => "super_administrator",
```

Comment:
I feel like when laying this out, then transient attributes should be separated or at the bottom so as not to mix into the real attributes (like first_name, last_name)